guess.js

```diff
@@ -50,6 +55,7 @@ function genrateInput() {
     inputsContainer.children[0].children[1].focus();
     //disabled All inpus expect frist input one
 
+    // [REVIEW] You could do all of the below while generating the inputs above to avoid this extra loop.

```

Comment:
you mean like line number 42 !!

Reply:
Yes, but it's not only the `keydown` event, but also [disabling inputs](https://github.com/mohamed-sabry-1789/guess-word-game/blob/main/guess.js#L53-L54) and [adding `input` listener](https://github.com/mohamed-sabry-1789/guess-word-game/blob/main/guess.js#L59-L70) can be done in this loop to avoid writing the other loop below.

Reply:
i am not understand

Reply:
I will submit another pull request to explain this part.